### DISCUSSION TITLE
custom Recipes for private server using kubejs

### DISCUSSION THREAD

**Discussion by sarr95** [TechReborn/TechReborn#2754] [Q&A]
can someone point me in the direction of the methods names for each machine process. i want to add recipes that use this mods machine. kubejs allows for custom method calls for creating custom recipes using other mods but i have no idea where to find the event names for the machines in this mod.

**Comment by Ayutac** [comment]
what exactly do you need? Can't you just add JSON files for custom recipes? all TR recipes, regardless of machine, are done via JSON (and a few restrictions on recycler in the config)
